docs/book/src/migration/legacy/v2vsv3.md

```diff
@@ -52,7 +52,7 @@ Projects scaffolded with Kubebuilder v3 will use the `go.kubebuilder.io/v3` plug
 <aside class="note warning">
 <h1>Project customizations</h1>
 
-After using the CLI to create your project, you are free to customise how you see fit. Bear in mind, that it is not recommended to deviate from the proposed layout unless you know what you are doing.
+After using the CLI to create your project, you are free to customize how you see fit. Bear in mind, that it is not recommended to deviate from the proposed layout unless you know what you are doing.
```

Comment:
We need to check I think amaerican english is z and bristh english is s so I am ok with the change